app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb

```diff
@@ -19,7 +19,7 @@ def start
     time = Time.zone.now
     update_attributes(:started_on => time)
     miq_task.update_attributes(:started_on => time)
-    my_signal(false, :launch_ansible_tower_job)
+    my_signal(false, :launch_ansible_tower_job, :priority => MiqQueue::HIGH_PRIORITY)
```

Comment:
can we default to normal priority, and accept an option to increase priority?

Reply:
Also while the start may need to be in high priority (caller is waiting to get a task handle), the retry/wait_for_completion can in normal priority.

Reply:
That is how it is implemented.

Reply:
I mean let the caller to decide the priority when call `start` or `create_job`, not hardcode the `HIGH_PRIORITY` here.